### PR TITLE
[OK] Pavel Yurchuk. Day 1 

### PR DESCRIPTION
**Pull request by pyurchuk**

### CODE REVIEW

File: day1/scripts/agent.sh [hunk 1]
```diff
@@ -1,3 +1,13 @@
 #!/bin/bash
 
-echo Installing Zabbix Agent
+sudo su
+yum -y install net-rools
```
Comment (sbeliakou) [inline day1/scripts/agent.sh:4]: why do we need this?

File: day1/scripts/server.sh [hunk 2]
```diff
@@ -1,3 +1,55 @@
 #!/bin/bash
 
-echo Installing Zabbix Server
+sudo su
+yum -y install mariadb mariadb-server
+/usr/bin/mysql_install_db --user=mysql
+systemctl start mariadb
+mysql -uroot -Bse "create database zabbix character set utf8 collate utf8_bin;
+grant all privileges on zabbix.* to zabbix@localhost identified by 'zabbix';"
```
Comment (sbeliakou) [inline day1/scripts/server.sh:8]: creds scattered everywhere

File: day1/scripts/server.sh [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+yum -y install net-tools
```
Comment (sbeliakou) [inline day1/scripts/server.sh:9]: why?

File: day1/scripts/server.sh [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+yum -y install http://repo.zabbix.com/zabbix/3.2/rhel/7/x86_64/zabbix-release-3.2-1.el7.noarch.rpm
+yum -y install zabbix-server-mysql zabbix-web-mysql
+zcat /usr/share/doc/zabbix-server-mysql-*/create.sql.gz | mysql -uzabbix -pzabbix zabbix
+
+sed -i '/# DBPassword=/ a\DBPassword=zabbix' /etc/zabbix/zabbix_server.conf
```
Comment (sbeliakou) [inline day1/scripts/server.sh:14]: again creds ...

File: day1/scripts/server.sh [hunk 5]
```diff
@@ -1,3 +1,55 @@
...
+<?php
+// Zabbix GUI configuration file.
+global \$DB;
+
+\$DB['TYPE']     = 'MYSQL';
+\$DB['SERVER']   = 'localhost';
+\$DB['PORT']     = '0';
+\$DB['DATABASE'] = 'zabbix';
+\$DB['USER']     = 'zabbix';
+\$DB['PASSWORD'] = 'zabbix';
```
Comment (sbeliakou) [inline day1/scripts/server.sh:30]: creds ...

File: day1/scripts/server.sh [hunk 6]
```diff
@@ -1,3 +1,55 @@
...
+	DocumentRoot /usr/share/zabbix
+	ServerName zabbix-server
+</VirtualHost>
+EOF
+
+yum -y install zabbix-agent
+systemctl start httpd
+systemctl enable zabbix-server
+systemctl start zabbix-server
+systemctl enable zabbix-server
```
Comment (sbeliakou) [inline day1/scripts/server.sh:53]: :)

File: day1/scripts/agent.sh (hunk 1 above)
Comment (PaulYurchuk) [inline day1/scripts/agent.sh]: yum -y install net-Tools - that was added for debugging

File: day1/scripts/server.sh (hunk 2 above)
Comment (PaulYurchuk) [inline day1/scripts/server.sh]: do you mean that is not secure to use credentials without encrypting?

File: day1/scripts/server.sh (hunk 6 above)
Comment (PaulYurchuk) [inline day1/scripts/server.sh]: lol

systemctl start httpd
systemctl enable httpd
systemctl start zabbix-server
systemctl enable zabbix-server

**Pull request closed without merging**